GitTfs/Core/IGitTfsRemote.cs

```diff
@@ -56,7 +56,7 @@ public interface IGitTfsRemote
         void QuickFetch();
         void QuickFetch(int changesetId);
         void Unshelve(string shelvesetOwner, string shelvesetName, string destinationBranch, Action<Exception> ignorableErrorHandler, bool force);
-        void Shelve(string shelvesetName, string treeish, TfsChangesetInfo parentChangeset, bool evaluateCheckinPolicies);
+        void Shelve(string shelvesetName, string head, TfsChangesetInfo parentChangeset, CheckinOptions options, bool evaluateCheckinPolicies);
```

Comment:
Yeah, let's leave it as `treeish`. I don't know how accurate that is now (i.e. does git-tfs really just need a tree, or is it interested in the whole commit?), but this may as well be consistent with the other definitions of `Shelve`.